### PR TITLE
Create Rule “using-digital-asset-manager/rule”

### PR DESCRIPTION
**Pull request by williamgreentreessw**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/using-digital-asset-manager/rule.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+type: rule
+title: Video Data - Do you use a Digital Asset Manager (DAM) like Adobe Bridge
+  to manage a stock library
```
Comment (tiagov8) [inline rules/using-digital-asset-manager/rule.md:4]: Missing question mark

File: rules/using-digital-asset-manager/rule.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+related:
+  - how-to-name-documents
+  - semantic-versioning
+  - organize-and-back-up-your-files
+created: 2023-08-09T02:20:28.506Z
+guid: 114547d6-0e8b-47aa-aee6-5e6b12b57212
+---
+## Why get a DAM?
+
+Trying to manage your large library of digital files can be pretty difficult using just Windows File Explorer or Mac's Finder as you are limited to folder structure and file names, which can lead to lost and abandoned assets due to ridged search functionality. When trying to preview your assets, your operating system may also lack support for previewing proprietary formats (E.G. Fujifilm's .RAF or Nikon's .NEF). A Good digital asset manager like Adobe Bridge solves this.
```
Comment (tiagov8) [inline rules/using-digital-asset-manager/rule.md:22]: 1. Change from "E.G." to "E.g."

2. Change from "Good" to "good"

File: rules/using-digital-asset-manager/rule.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+while there are several digital asset managers on the market, each with its strengths and weaknesses, for the majority of small teams, independent creatives and the budget-conscious, Adobe Bridge is more than enough to handle the task.
+
+## Why do you need a stock library?
+
+When video editing, you might need a certain shot that was not captured on production due to oversite or budget constrictions. That is why editors turn to stock footage libraries like the following:
+
+* https://www.filmsupply.com
```
Comment (tiagov8) [inline rules/using-digital-asset-manager/rule.md:30]: Can we make these links friendlier for readers?

E.g.

from
`https://www.pexels.com`

To
`pexels.com`

File: rules/using-digital-asset-manager/rule.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+While stock footage sites are great for getting high-quality footage for your project, they lack the personal touch of your company or brand. That is why it is important to build your own internal stock library. While this does add extra effort to each project you shoot to build your library, the benefits of doing so include:
+
+* Having footage that includes your logos.
+* Showing off the people that work with you.
+* Copyright ownership of the footage.
+
+Best practice for creating your stock library. As soon as you are finished with your video project. 
+
+1. You should open a new timeline and drop all the B-Roll clips that you have collected into that timeline.
```
Comment (tiagov8) [inline rules/using-digital-asset-manager/rule.md:47]: Can you add a link to the B-roll rule on "B-roll" word?

File: rules/using-digital-asset-manager/rule.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+* https://artlist.io
+* https://stock.adobe.com
+* https://motionarray.com
+* https://elements.envato.com
+* https://pixabay.com
+* https://www.pexels.com
+
+While stock footage sites are great for getting high-quality footage for your project, they lack the personal touch of your company or brand. That is why it is important to build your own internal stock library. While this does add extra effort to each project you shoot to build your library, the benefits of doing so include:
+
+* Having footage that includes your logos.
```
Comment (tiagov8) [inline rules/using-digital-asset-manager/rule.md:41]: Please follow https://www.ssw.com.au/rules/avoid-full-stops-in-bullet-point-lists/ on your lists

File: rules/using-digital-asset-manager/rule.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+From there you can use a digital asset manager like Adobe Bridge to add further metadata information to help with navigating your personal stock footage library.
+
+## What is Adobe Bridge?
+
+Adobe Bridge is a creative digital asset manager that lets you preview, organize, edit, and publish multiple creative assets (including Adobe Photoshop, InDesign, Illustrator, After Effects, and Dimension files) with thumbnails and rich previews. Edit metadata. Add keywords, labels, and ratings to assets. Organize assets using collections, and find assets using powerful filters and advanced metadata search features.
+
+## Organize your media files
+
+By creating a well-structured folder hierarchy and sticking to a standard naming convention, we can make sure media assets won't get accidentally deleted or go missing.
+Consider organizing videos based on projects, clients, or types of footage (e.g., interviews, B-roll, aerial shots).
```
Comment (tiagov8) [inline rules/using-digital-asset-manager/rule.md:63]: From "e.g.," to "E.g."

File: rules/using-digital-asset-manager/rule.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+Adobe Bridge is a creative digital asset manager that lets you preview, organize, edit, and publish multiple creative assets (including Adobe Photoshop, InDesign, Illustrator, After Effects, and Dimension files) with thumbnails and rich previews. Edit metadata. Add keywords, labels, and ratings to assets. Organize assets using collections, and find assets using powerful filters and advanced metadata search features.
+
+## Organize your media files
+
+By creating a well-structured folder hierarchy and sticking to a standard naming convention, we can make sure media assets won't get accidentally deleted or go missing.
+Consider organizing videos based on projects, clients, or types of footage (e.g., interviews, B-roll, aerial shots). 
+
+Learn more on:
+
+* [Video data - Do you back up your files in an organized way?](https://www.ssw.com.au/rules/organize-and-back-up-your-files/)
```
Comment (tiagov8) [inline rules/using-digital-asset-manager/rule.md:67]: You can simplify internal links to other rules...
From
(https://www.ssw.com.au/rules/organize-and-back-up-your-files/)

to
(/organize-and-back-up-your-files)

File: rules/using-digital-asset-manager/rule.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+2. Next, click on export to export your current keyword list.
+
+![Figure: Export keywords list](export-scr.png "Figure: Export keywords list")
+
+3. Save this file with a version number to a shared folder accessible to everyone in your team.
+4. To import an existing keyword list, navigate and click on import from the same list. Then select the .XML file you want to import.
+
+![Figure: Import keywords list](import-scr.png "Figure: Import keywords list")
+
+Know more on:
```
Comment (tiagov8) [inline rules/using-digital-asset-manager/rule.md:95]: You already have these 2 rules on the Related list... so this last section is not necessary...

File: rules/using-digital-asset-manager/rule.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
+---
+type: rule
+title: Video Data - Do you use a Digital Asset Manager (DAM) like Adobe Bridge
+  to manage a stock library
+uri: using-digital-asset-manager-for-stock
+authors:
+  - title: William Fardell
+    url: https://ssw.com.au/people/alumni/will/
+    img: https://ssw.com.au/people/static/Will-Greentree-Profile-5b47d1b6e1f91074e10546d28020eb19.jpg
+  - title: SSW RulesGPT 3.5 turbo
```
Comment (tiagov8) [inline rules/using-digital-asset-manager/rule.md:10]: As per our conversation and your question to Uly, please remove this author

File: rules/using-digital-asset-manager/rule.md (hunk 1 above)
Comment (williamgreentreessw) [reply rules/using-digital-asset-manager/rule.md:4]: Done

File: rules/using-digital-asset-manager/rule.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
+---
+type: rule
+title: Video Data - Do you use a Digital Asset Manager (DAM) like Adobe Bridge
```
[the remark this answers is not in the document]
Comment (williamgreentreessw) [reply rules/using-digital-asset-manager/rule.md:3]: done

File: rules/using-digital-asset-manager/rule.md (hunk 3 above)
Comment (williamgreentreessw) [reply rules/using-digital-asset-manager/rule.md:27]: Done

File: rules/using-digital-asset-manager/rule.md (hunk 2 above)
Comment (williamgreentreessw) [reply rules/using-digital-asset-manager/rule.md:19]: done

File: rules/using-digital-asset-manager/rule.md (hunk 4 above)
Comment (williamgreentreessw) [reply rules/using-digital-asset-manager/rule.md:44]: done - rule used https://www.ssw.com.au/rules/video-editing-terms/#4-b-roll-footage

File: rules/using-digital-asset-manager/rule.md (hunk 8 above)
Comment (williamgreentreessw) [reply rules/using-digital-asset-manager/rule.md:92]: Done

File: rules/using-digital-asset-manager/rule.md (hunk 9 above)
Comment (williamgreentreessw) [reply rules/using-digital-asset-manager/rule.md]: Done

File: rules/using-digital-asset-manager/rule.md (hunk 6 above)
Comment (williamgreentreessw) [reply rules/using-digital-asset-manager/rule.md:60]: Done

File: rules/using-digital-asset-manager/rule.md (hunk 7 above)
Comment (williamgreentreessw) [reply rules/using-digital-asset-manager/rule.md:64]: Done - Dropped link as it is linked in the related

**Review by UlyssesMaclaren — APPROVED** [review]

**Pull request merged**